packages/config/src/doc/IProfiInfoProps.ts

```diff
@@ -0,0 +1,29 @@
+/*
+* This program and the accompanying materials are made available under the terms of the
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Copyright Contributors to the Zowe Project.
+*
+*/
+
+/**
+ * Options that will affect the behavior of the ProfileInfo class.
+ * They are supplied on the ProfileInfo getAllProfiles method.
+ */
+ export interface IGetAllProfilesOptions {
+
+  /**
+   * Indicates if the home directory should be excluded
+   * The default is false.
+   */
+   excludeHomeDir?: boolean;
```

Comment:
Passing `excludeHomeDir` on the getAllProfiles method makes sense if it meets the use case 👍 Another option we could consider is an enum option on the readProfilesFromDisk method to `"ignore" | "fallback" | "merge"` the home dir.

Reply:
Discussed this idea and decided against it, as it seems unnecessary and could be confusing. As long as we maintain the desired behavior of the ProfileInfo API in Zowe Explorer, I'm good with the current implementation 🙂

Reply:
I will consider these sort of comments whenever we need to implement some sort of binary implementation incase there is more than just two routes we can take 👍 
Enums are cool too 😋